toc.hbs.md

```diff
@@ -616,6 +616,7 @@
         - [Types of Vulnerability Scans](scst-scan/scan-types.hbs.md)
         - [Install Supply Chain Security Tools - Scan](scst-scan/install-scst-scan.hbs.md)
         - [Upgrade Supply Chain Security Tools - Scan](scst-scan/upgrading.hbs.md)
+        - [Upgrade Supply Chain Security Tools - AMR BETA](scst-scan/upgrading-amr-beta.hbs.md)
```

Comment:
There're two mistakes here.

This is in the wrong section of the TOC. It is in the Scan section when it should be in the Store section.

The link is also incorrectly pointing to `scst-scan` folder.